server/services/third-party-apis/eff-aws-ses.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Helpers for working with the EFF Civic CRM APIs.
+ */
+
+var path = require('path');
+var url = require('url');
+
+// load aws sdk
+var aws = require('aws-sdk');
+
+var emailUserCopyOfMessage = function(params, config, cb) {
+
+  awsConfig = config.get('CREDENTIALS.AWS');
+
+
+  // load aws config
+  aws.config.update(awsConfig);
+
+  // load AWS SES
+  var ses = new aws.SES({apiVersion: '2010-12-01'}); //TODO is this version the best one?
+
+  // send to list
+  var to = [params.contact_params.email];
+
+  // this must relate to a verified SES account
+  var from = 'example@example.com'; //TODO change to EFF email verified with AWS SES
```

Comment:
If we do want to put it in the Config, I'll defer to you all as the best place to put it. Doesn't neatly fit under Credentials or API